pkgs/by-name/ma/mailsy/package.nix

```diff
@@ -0,0 +1,32 @@
+{
+  buildNpmPackage,
+  fetchFromGitHub,
+  lib,
+}:
+buildNpmPackage rec {
+  pname = "mailsy";
+  version = "5.0.0";
+
+  src = fetchFromGitHub {
+    owner = "BalliAsghar";
+    repo = "Mailsy";
+    rev = version;
+    hash = "sha256-RnOWvu023SOcN83xEEkYFwgDasOmkMwSzJ/QYjvTBDo=";
+  };
+
+  npmDepsHash = "sha256-ljmqNmLvRHPdsKyOdDfECBXHTIExM6nPZF45lqV+pDM=";
+
+  npmFlags = ["--legacy-peer-deps"];
```

Comment:
The file doesn't seem to be properly formatted. You should format it with [nixfmt](https://github.com/NixOS/nixfmt)